exotic/api/nea.py

```diff
@@ -220,10 +221,13 @@ def _new_scrape(self, filename="eaConf.json"):
         if os.path.exists('pl_names.json'):
             with open("pl_names.json", "r") as f:
                 planets = json.load(f)
-                for key, value in planets.items():
-                    if self.planet.lower().replace(' ', '').replace('-', '') == key:
-                        self.planet = value
-                        break
+                planet_key = re.sub(r'[^a-zA-Z0-9]', '', self.planet.lower())
```

Comment:
A comment here would be appreciated. This is basically removing all non-alphanumerics (including spaces) and lowercasing target names. This may be a better approach than used currently in CITISENS, not sure, but I'm also not convinced yet that all target aliases are accounted for here. _Nothing to change, maybe add comment._

Reply:
@tamimfatahi